src/fonda/execute.cljs

```diff
@@ -3,6 +3,20 @@
             [fonda.async :as a]
             [fonda.step :as st]))
 
+(defn get-callback-fn
+  [{:as fonda-ctx :keys [callbacks-wrapper-fn]} callback-val-or-fn]
```

Comment:
How about we throw exception here when the `callback-val-or-fn` is  a function and `callbacks-wrapper-fn` is  also provided? The fact that this is possible is going to allow the consumers of library to see unexpected result. Why not prevent this on library itself?

Reply:
This allows you to pass a function if you want. So you can provide values on certain callbacks, and functions on others. I'm finding it useful already not only for debugging but also for logging.
What I think that should throw an exception is when a value is provided, and no `callback-val-or-fn` is passed.
Maybe we can discuss it further if you don't think it makes sense